crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -153,19 +150,35 @@ impl DataTransfer {
         Ok(())
     }
 
-    /// Transfer all compressed files currently in the data folder to the remote object store.
-    /// Return [`Ok`] if all files were transferred successfully, otherwise [`ParquetError`]. Note
-    /// that if the function fails, some of the compressed files may still have been transferred.
-    /// Since the data is transferred separately for each table, the function can be called again if
-    /// it failed.
-    pub(crate) async fn flush(&self) -> Result<(), ParquetError> {
+    /// Set the transfer batch size to `new_value`. For each table that compressed data is saved
+    /// for, check if the amount of data exceeds `new_value` and transfer if so. If the value
```

Comment:
Maybe "... exceeds `new_value` and transfer data if it is." Also maybe document how much data is transferred, is it all of the data or just enough to get below `new_value`.

Reply:
Fixed.